package.json

```diff
@@ -94,5 +94,8 @@
     "sinon": "^1.17.4",
     "sinon-as-promised": "^4.0.0",
     "sinon-chai": "^2.8.0"
+  },
+  "optionalDependencies": {
```

Comment:
SGTM. Given that, can you please add a comment above the require for the emulator that states that this **must** be required during runtime in a try/catch block since it's an optional dependency?